ci/jenkins/test-vmc.sh

```diff
@@ -323,14 +336,42 @@ function deliver_antrea {
         scp -q -o StrictHostKeyChecking=no -i ${GIT_CHECKOUT_DIR}/jenkins/key/antrea-ci-key $GIT_CHECKOUT_DIR/build/yamls/*.yml capv@${master_ip}:~
     done
 
-    kubectl get nodes -o wide --no-headers=true | awk '{print $6}' | while read IP; do
-        ssh-keygen -f "/var/lib/jenkins/.ssh/known_hosts" -R ${IP}
+    IPs=($(kubectl get nodes -o wide --no-headers=true | awk '{print $6}' | xargs))
+    for i in "${!IPs[@]}"
+    do
+        ssh-keygen -f "/var/lib/jenkins/.ssh/known_hosts" -R ${IPs[$i]}
         if [[ "$COVERAGE" == true ]]; then
-            copy_image antrea-ubuntu-coverage.tar docker.io/antrea/antrea-ubuntu-coverage $IP ${DOCKER_IMG_VERSION}
+            copy_image antrea-ubuntu-coverage.tar docker.io/antrea/antrea-ubuntu-coverage ${IPs[$i]} ${DOCKER_IMG_VERSION} true
         else
-            copy_image antrea-ubuntu.tar projects.registry.vmware.com/antrea/antrea-ubuntu $IP ${DOCKER_IMG_VERSION}
+            copy_image antrea-ubuntu.tar projects.registry.vmware.com/antrea/antrea-ubuntu ${IPs[$i]} ${DOCKER_IMG_VERSION} true
+        fi
+        copy_image flow-aggregator.tar projects.registry.vmware.com/antrea/flow-aggregator ${IPs[$i]} ${DOCKER_IMG_VERSION} true
+    done
+
+    if [[ -z $OLD_ANTREA_VERSION ]]; then
+        return 0
+    fi
+
+    echo "====== Pulling old Antrea images ======"
+    if [[ ${DOCKER_REGISTRY} != "" ]]; then
+        docker pull ${DOCKER_REGISTRY}/antrea/antrea-ubuntu:$OLD_ANTREA_VERSION
+    else
+        docker pull antrea/antrea-ubuntu:$OLD_ANTREA_VERSION
+        docker tag antrea/antrea-ubuntu:$OLD_ANTREA_VERSION projects.registry.vmware.com/antrea/antrea-ubuntu:$OLD_ANTREA_VERSION
+    fi
+
+    # Deliver old-versioned Antrea images
```

Comment:
Maybe remove this comment since it duplicates with the echo message below.

Reply:
Deleted.